src/pages/landing/HomePage.vue

```diff
@@ -17,6 +19,7 @@ export default defineComponent({
   <HomeHeroBanner class="hero" />
   <HomeMoviesPanel class="movies" />
   <ScreeningsPanel heading-size="medium" class="screenings" />
+  <HomeContactPanel class="contact" />
```

Comment:
Unless this `class` override other styles that are specified inside that component consider encapsulating it within that component.

Reply:
Makes sense, will do 👍